src/components/VideoPlayer/PlayButton.tsx

```diff
@@ -0,0 +1,74 @@
+import { FC } from "react";
+import styled, { css } from "styled-components";
+
+import { Icon } from "../Icon";
+
+export interface PlayButtonProps {
+  isPlaying?: boolean;
+  onClick?: () => void;
+  testId?: string;
+}
+
+const PlayButtonRoot = styled.div<Pick<PlayButtonProps, "isPlaying">>`
+  position: relative;
+  display: flex;
+  height: 80px;
+  width: 80px;
+  align-items: center;
+  justify-content: center;
+  border: 3px solid ${(props) => props.theme.color.white};
+  border-radius: 50px;
+  background-color: ${(props) => props.theme.color.primary};
+  box-shadow: ${(props) => props.theme.videoPlayer.buttonBoxShadow};
+  cursor: pointer;
+  overflow: hidden;
+
+  transition: all 0.3s ease-out;
+
+  ${(props) =>
+    props.isPlaying &&
+    css`
+      opacity: 0;
+    `}
+`;
+
+const InternalBorder = styled.div`
+  box-sizing: border-box;
+  height: 100%;
+  width: 100%;
+  border: 3px solid ${(props) => props.theme.color.white};
+  opacity: 0.4;
+  border-radius: 50px;
+  z-index: 1;
+
+  position: absolute;
+`;
+
+const IconWrapper = styled.div`
```

Comment:
In questi casi si può fare lo stile direttamente sul componente Icon, ti risparmi un wrapper